src/Core/Components/DataGrid/Columns/ColumnBase.razor

```diff
@@ -20,7 +20,16 @@
             @if (ColumnOptions is not null && (Align == Align.Start || Align == Align.Center))
             {
                 <FluentButton Appearance="Appearance.Stealth" class="col-options-button" @onclick="@(() => Grid.ShowColumnOptionsAsync(this))" aria-label="Filter this column">
-                    <FluentIcon Value="@(new CoreIcons.Regular.Size24.Filter())" />
+                    @if(Filtered.GetValueOrDefault())
```

Comment:
I find this way of writing clearer for testing the same thing.  What do you think?

```csharp
@if(Filtered == true)
```

Reply:
I agee will change that